qt/scientific_interfaces/Indirect/ISISEnergyTransfer.cpp

```diff
@@ -273,9 +273,7 @@ void ISISEnergyTransfer::run() {
       createMapFile(m_uiForm.cbGroupingOptions->currentText());
```

Comment:
I get a compilation error here concerning type conversion between QString and const std::string.